### PR TITLE
Feature/issue 168 ordering and paging

### PR DESCRIPTION
**Pull request by rasmus-storjohann-PG**

### CODE REVIEW

File: services/tests/test_search.py [hunk 1]
```diff
@@ -289,34 +292,36 @@ def test_can_specify_page_size(self):
...
+        self.assertEqual(len(response.json()), 30)
+
+    def test_clamps_max_items_per_page_to_100(self):
+        self.create_many_services(101)
+
+        response = self.client.get('/v1/services/?per_page=101')
+
+        self.assertEqual(len(response.json()), 100)
+
+    def test_page_number_defaults_to_one(self):
```
Comment (bensinjin) [inline services/tests/test_search.py]: What do you mean by test page number defaults to one? To me this test looks like it's checking the services json contains the expected ids and number of records?

**Comment by bensinjin** [comment]
This is looking good to me. I haven't seen anything glaring.

**Comment by rasmus-storjohann-PG** [comment]
Thanks @bensinjin , fixed the tests that weren't clear.

**Comment by bensinjin** [comment]
Much better, nice.

**Pull request merged**